src/wallet/rpcwallet.cpp

```diff
@@ -4665,15 +4665,14 @@ size_t EstimateTxSize(
                 jsdesc.proof = GrothProof();
                 mtx.vJoinSplit.push_back(jsdesc);
             },
-            [&](const libzcash::OrchardRawAddress& addr) {
-                if (fromSprout) {
-                    throw JSONRPCError(
-                        RPC_INVALID_PARAMETER,
-                        "Sending funds from a Sprout address to a Unified Address is not supported by z_sendmany");
+            [&](const libzcash::UnifiedAddress& addr) {
+                if (addr.GetOrchardReceiver().has_value()) {
+                    orchardRecipientCount += 1;
+                } else if (addr.GetSaplingReceiver().has_value()) {
+                    mtx.vShieldedOutput.push_back(OutputDescription());
                 }
```

Comment:
This should probably also have an `else if` case for resolving to a transparent receiver.